lib/service/signals.go

```diff
@@ -134,6 +133,37 @@ func (process *TeleportProcess) WaitForSignals(ctx context.Context) error {
 	}
 }
 
+func (process *TeleportProcess) writeToSignalPipe(message string) error {
+	signalPipe, err := process.importSignalPipe()
+	if err != nil {
+		if !trace.IsNotFound(err) {
+			return trace.Wrap(err)
+		}
+		log.Debugf("No signal pipe found, not going to signal")
```

Comment:
Maybe change message to, "No signal pipe to import, must be first Teleport process."